docs/fidesops/docs/css/fides.css

```diff
@@ -18,9 +18,16 @@
     --md-code-hl-string-color: rgb(133, 217, 150);
     --md-code-hl-keyword-color: rgb(245, 251, 255);
     --md-code-hl-punctuation-color: rgb(245, 251, 255);
-    --md-bg-footer-color: #0861CE;
+    --md-code-hl-number-color:  #F8B886;
+    --md-code-hl-special-color: #F8B886;
+    --md-bg-footer-color: #111439;
     --md-bg-table-header: rgba(79, 86, 107, 0.05);
     --svg-highlight-col: #0657b9;
+    --code-window-bg:  #2A2F45;
```

Comment:
just curious, is there any reasoning behind using hex in some places vs rgb in others?

Reply:
When I started working on this there was already a mix of rgb and hex used, I tried to stick with it whatever format it was already done in when adjusting the values, but defaulted to hex when adding new values as it's what we use for the brand site/guidelines and its easier for me to recognize. I can go back and re-do the values in rgba if that's preferred!

Reply:
no worries, what you did is fine for this PR!